### DISCUSSION TITLE
layout of BarButtonItems issue

### DISCUSSION THREAD

**Discussion by law-cell** [LeoNatan/LNPopupController#455] [General] [answered]
when I set images of trailingBarButtonItems and leadingBarButtonItems
this happened
![Simulator Screen Shot - iPhone 11 - 2021-04-22 at 22 05 19](https://user-images.githubusercontent.com/59221817/115728187-dc47cb00-a3b6-11eb-850d-ae16d90ba745.png)

when I set one of two buttonitems
the position of the button looks good
![simulator_screenshot_57083D69-9FBB-40B1-BF25-A5C37F32A6B8](https://user-images.githubusercontent.com/59221817/115728381-0dc09680-a3b7-11eb-8f98-69556d6d0027.png)

#### Steps to Reproduce

func setMiniPlayer(){
        let storyboard = UIStoryboard(name: "Main", bundle: nil)
        let demoVC = storyboard.instantiateViewController(withIdentifier: "playerVC")
        
        self.popupNextButtonItem = UIBarButtonItem(image: UIImage(named: "next"), style: .plain, target: nil, action: nil)
        self.popupPlayButtonItem = UIBarButtonItem(image: UIImage(named: "play.fill"), style: .plain, target: nil, action: nil)
        
        
        self.musicImage = UIImage.init(named: "genre1")
        
        self.popupBar.barStyle = .prominent
        demoVC.popupItem.title = "Hello World"
        demoVC.popupItem.subtitle = "And a subtitle!"
        //demoVC.popupItem.progress = 0.34
        demoVC.popupItem.trailingBarButtonItems = [self.popupNextButtonItem!]
        demoVC.popupItem.leadingBarButtonItems = [self.popupPlayButtonItem!]
        demoVC.popupItem.image = musicImage
        
        self.presentPopupBar(withContentViewController: demoVC, animated: true, completion: nil)
    }

#### Device, OS and Xcode Versions
ios 13
iPhone 11

**Comment by LeoNatan** [comment, marked as answer]
It looks like your button images might be too big. Are you using images from your catalog? Instead, try using `UIImage(systemName: "...")` instead.
